Web_Development/Javascript/Topics/5.Syntax/2.Operators/ArithmeticOperator.md

```diff
@@ -0,0 +1,116 @@
+# Arithmetic Operators
+[Back to Operators in Javascript](README.md#operators-in-javascript)
+
+#### 1. Addition (+)
```

Comment:
First of all Make ArithmeticOperator.md in it's folder named `1.Arithmetic_Operators` and rename the file to *README.md*